airflow/migrations/versions/c2e6d7e1c6d2_merge_61ec_and_e1a.py

```diff
@@ -0,0 +1,44 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Merge 61ec and e1a
```

Comment:
Let's update the other migration in place since this PR is not merged instead of merging the heads

Reply:
Done. Waiting for the tests to finish running.

Reply:
@kaxil some checks were skipped although nothing else failed, not sure why.

Reply:
@kaxil can you please review? Would be great if we can close this soon.